ledger-tool/src/main.rs

```diff
@@ -1246,6 +1246,8 @@ fn main() {
         )
         .get_matches();
 
+    info!("{} {}", crate_name!(), solana_version::version!());
```

Comment:
Do we need this log?

Reply:
@mvines Yeah, this is useful for developing/bisecting. When I finds out that something bad started to occur, I want to know the exact commit hash, which started the odd behavior. I tend to `(./target/release/solana-ledger-tool --version && ./target/release/solana-ledger-tool ...) &> /path/to/log`. And it's tedious at least. Also, `solana-validator` does already this, why not for `solana-ledger-tool`?

Reply:
Got it, thanks for the explanation